packages/web-components/fast-foundation/src/tooltip/tooltip.template.ts

```diff
@@ -10,17 +10,17 @@ export const TooltipTemplate = html<Tooltip>`
         x => x.tooltipVisible,
         html<Tooltip>`
             <fast-anchored-region
-                ${ref("region")}
-                dir=${x => x.currentDirection}
-                vertical-positioning-mode=${x => x.verticalPositioningMode}
-                vertical-default-position=${x => x.verticalDefaultPosition}
-                vertical-inset=${x => x.verticalInset}
-                vertical-scaling=${x => x.verticalScaling}
-                horizontal-positioning-mode=${x => x.horizontalPositioningMode}
-                horizontal-default-position=${x => x.horizontalDefaultPosition}
-                horizontal-scaling=${x => x.horizontalScaling}
-                horizontal-inset=${x => x.horizontalInset}
                 fixed-placement="true"
+                vertical-positioning-mode="${x => x.verticalPositioningMode}"
+                vertical-default-position="${x => x.verticalDefaultPosition}"
+                vertical-inset="${x => x.verticalInset}"
+                vertical-scaling="${x => x.verticalScaling}"
+                horizontal-positioning-mode="${x => x.horizontalPositioningMode}"
+                horizontal-default-position="${x => x.horizontalDefaultPosition}"
+                horizontal-scaling="${x => x.horizontalScaling}"
+                horizontal-inset="${x => x.horizontalInset}"
+                dir="${x => x.currentDirection}"
```

Comment:
Should we have tests for these as they're technically part of the tooltip API? That might help catch a regression here in the future.

Reply:
Yes, I will be boosting tooltip and anchored region test coverage as it is currently bare bones, although I don't think we can write a test to catch this particular issue as the only repro so far is in the doc environment. If it turns out that component attribute bindings should be wrapped in quotes generally then maybe that's something we can test for, but I don't think we've decided to make that a requirement yet.